search-app/src/cmr/search/site/routes.clj

```diff
@@ -24,22 +24,29 @@
              (pages/home ctx))
         (GET "/sitemap.xml"
              {ctx :request-context}
-             (pages/sitemap-master ctx))
-        (GET "/site/sitemap.xml"
-             {ctx :request-context}
-             (pages/sitemap-top-level ctx))
+             {:status 200
+              :body (slurp (static/site-resource "../sitemap.xml"))})
         (GET "/site/collections/directory"
              {ctx :request-context}
              (pages/collections-directory ctx))
         (GET "/site/collections/directory/eosdis"
-             {ctx :request-context}
-             (pages/eosdis-collections-directory ctx))
+             [provider-id tag]
+             (when-let [resource (static/site-resource
+                                  (format
```

Comment:
copy paste error here. Should be just "collections/directory/eosdis/index.html".

Reply:
It's actually not a typo -- dirs automatically render their index files for the directory pages now.